wpf/README.md

```diff
@@ -19,7 +19,7 @@ See [WPF Samples](https://www.github.com/Microsoft/wpf-samples) repo for additio
 
 ### Prerequisites and getting the tools
 
-Install Visual Studio 2017 Update 15.8 or higher from [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/), selecting the **.NET desktop development** workload with the options: **.NET Framwork 4.7.2 development tools** and **.NET Core 2.1 development tools**.
+Install Visual Studio 2019 preview from [https://visualstudio.microsoft.com/vs/preview](https://visualstudio.microsoft.com/vs/preview), selecting the **.NET desktop development** workload with the options: **.NET Framwork 4.7.2 development tools** and **.NET Core 2.2 development tools**. 
```

Comment:
@dsplaisted  do we still need 4.7.2 development tools for WPF development on Core 3 preview 2?

Reply:
I don't think so.  I think the reason we needed it before was because we were using the WPF targets (PresentationBuildTasks) from the full Framework, which we don't do by default anymore.